content/c/concepts/functions/functions.md

```diff
@@ -0,0 +1,80 @@
+---
+Title: "Functions"
+Subjects: 
+  - "Code Foundations"
+  - "Computer Science"
+Tags:
+  - "Functions"
+---
+
+Functions are blocks of code that can be run multiple times within a program. The are stored under a name similar to a variable and can be run using this name.
+
+## Declaring a Variable
+
+A function is declared with the return type, the name,  parenthesis, and the code for it to run in curly brackets. 
+
+```pseudo
+return_type name(parameters) {
+  line_of_code;
+  line_of_code;
+}
+```
+
+A variable that prints a greeting to the console may look like this.
+
+```c
+void greet(void) {
+  puts("Howdy!\n");
+}
+```
+
+## Calling a Function
+
+After declaring a function it can be used in a program by calling it. The sytax for this is the variables name followed by parenthesis.
+
+```c
+#include <stdio.h>
+
+void greet(void) {
+  puts("Howdy!\n");
+}
+
+int main(void) {
+  greet(); // prints Howdy!
+}
+```
+
+## Arguments 
+
+A function may need to do more than one thing based on some condition, and for this are arguments. Arguments are variables passed to the function when it is called that can be used in it. 
+They are set inside of the functions parenthesis when it is declared in the same way as a variable, and values are passed to the function inside the parenthesis when it is called.
+
+```c
+#include <stdio.h>
+
+void greet(char* name) {
+  printf("Howdy %s!\n")
+}
+
+int main(void) {
+  greet("John Doe"); // prints Howdy John Doe!
```

Comment:
I would write the comment like: `// Output: Howdy John Doe!` to have a consistency with rest of the Codecademy Docs entries.

Reply:
Done